samples/markdown_separators.md

```
@@ -0,0 +1,97 @@
+  toBeat?: (md: string) => Beat | null;
+}
+```
+
+Code blocks are preserved in markdown output.
+
+---
+
+# Table Example
+
+| Separator | Flag | Description |
+|-----------|------|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)